plugins/modules/ntnx_promote_protected_resources_v2.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Nutanix
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: ntnx_promote_protected_resources_v2
+short_description: Module to promote a protected resource in Nutanix Prism Central.
+description:
+  - This module can be used to promote a protected resource in Nutanix Prism Central.
+  - Supported only for protected resources that have synchronous protection policies.
+  - Promote VM will create new VM in the secondary PC for the given VM external ID
+  - promote VG will create new VG in the second local PE for the given VG external ID
+  - Promote VM uses secondary PC IP where Promote VG uses primary PC IP
+options:
+  ext_id:
+    description:
+      - The external identifier of a protected VM or volume group.
+    type: str
+    required: true
+  wait:
+    description:
+      - Wait for the task to complete.
+    type: bool
+    required: false
+    default: True
```

Comment:
Override docs for nutanix_host nutanix_username and nutanix_password attributes here. Doc in ntnx_operations_v2 can be misleading for this module.